### PR TITLE
Fixed issues with Applications page

### PR DESCRIPTION
**Pull request by clackner-gpa**

### CODE REVIEW

File: Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs [hunk 1]
```diff
@@ -48,6 +48,48 @@
 
 namespace SystemCenter.Controllers.OpenXDA
 {
+    public class OpenXDAApiController
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs:51]: I don't think this is necessary. It looks like this Class is only used in Line 515 for a single Get Call. See comments there, but I would remove this class.

File: Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs [hunk 2]
```diff
@@ -470,17 +512,7 @@ public class MeterDataQualitySummaryController : ModelController<MeterDataQualit
     public class RemoteXDAInstanceController : ModelController<RemoteXDAInstance>
     {
         #region [Properties]
-
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs:473]: I would keep this section in here.

File: Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs [hunk 3]
```diff
@@ -491,7 +523,7 @@ public IHttpActionResult TestRemoteXDAConnection(int remoteInstanceID)
                 return Unauthorized();
             try
             {
-                Task<string> responseTask = SendGetRequest($"/api/DataPusher/TestConnection/{remoteInstanceID}");
+                Task<string> responseTask = XdaApiController.SendGetRequest($"/api/DataPusher/TestConnection/{remoteInstanceID}");
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs:526]:
```suggestion
                Task<string> responseTask = SendGetRequest($"/api/DataPusher/TestConnection/{remoteInstanceID}");
```

File: Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs [hunk 4]
```diff
@@ -508,7 +540,7 @@ public virtual IHttpActionResult PushRemoteConfig(int remoteInstanceID)
                 return Unauthorized();
             try
             {
-                Task<string> responseTask = SendGetRequest($"/api/DataPusher/SyncInstanceConfig/SystemCenter/{remoteInstanceID}");
+                Task<string> responseTask = XdaApiController.SendGetRequest($"/api/DataPusher/SyncInstanceConfig/SystemCenter/{remoteInstanceID}");
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs:543]:
```suggestion
                Task<string> responseTask = SendGetRequest($"/api/DataPusher/SyncInstanceConfig/SystemCenter/{remoteInstanceID}");
```

File: Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs [hunk 5]
```diff
@@ -517,36 +549,33 @@ public virtual IHttpActionResult PushRemoteConfig(int remoteInstanceID)
             }
         }
         #endregion
+    }
 
-        private async Task<string> SendGetRequest(string requestURI)
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs:521]: Keep this.

File: Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs (hunk 1 above)
Comment (gcsantos-gpa) [reply Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs:51]: It is also used in line 558 in a separate controller

Comment (clackner-gpa) [reply Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs:51]: I'd still move this locally into the controller. (a) it's easier to track what settings are used if they are directly in the controller using them and (b) it's really the `API` that should contain these methods and not a separate class only existing in SystemCenter if we were to move that way.

**Pull request merged**